### PR TITLE
16621 implemented spinner during search

### PR DESCRIPTION
**Pull request by severinbeauvais**
*Issue #:* https://github.com/bcgov/entity/issues/16621

*Description of changes:*
- app version = 2.2.1
- appended spinner during searching

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the sbc-auth license (Apache 2.0).

### CODE REVIEW

File: auth-web/src/components/auth/manage-business/BusinessLookup.vue [hunk 1]
```diff
@@ -7,7 +7,7 @@
       :loading="state === States.SEARCHING"
       :name="Math.random()"
       :search-input.sync="searchField"
-      :append-icon="(state === States.SEARCHING) ? 'mdi-timer-sand' : 'mdi-magnify'"
+      append-icon="mdi-magnify"
```
Comment (severinbeauvais) [inline auth-web/src/components/auth/manage-business/BusinessLookup.vue:10]: This icon is displayed unless the one in the slot, below, overrides it.

File: auth-web/src/components/auth/manage-business/BusinessLookup.vue [hunk 2]
```diff
@@ -22,6 +22,16 @@
...
     >
+      <template v-slot:append>
+        <v-progress-circular
+          v-if="state === States.SEARCHING"
+          color="primary"
+          indeterminate
+          :size="24"
+          :width="2"
+        />
+      </template>
```
Comment (severinbeauvais) [inline auth-web/src/components/auth/manage-business/BusinessLookup.vue:33]: This slot is the exact size and location as the `append-icon` property above.

Icon under normal scenarios:

image

Icon during search (it's animated so you only see an arc segment here):

image

**Review by JazzarKarim — APPROVED** [review]

**Review by seeker25 — APPROVED** [review]

**Comment by pwei1018** [comment]
Temporary Url for review: https://bcregistry-account-dev--pr-2370-mmx0chvi.web.app

**Pull request merged**